src/wp-admin/widgets-form-blocks.php

```diff
@@ -24,7 +24,37 @@
 );
 block_editor_rest_api_preload( $preload_paths, $block_editor_context );
 
-$editor_settings = get_block_editor_settings( array(), $block_editor_context );
+// Editor Styles.
+$styles = array(
+	array(
+		'css'            => 'body { font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif }',
+		'__unstableType' => 'core',
+	),
+);
+if ( $editor_styles && current_theme_supports( 'editor-styles' ) ) {
+	foreach ( $editor_styles as $style ) {
+		if ( preg_match( '~^(https?:)?//~', $style ) ) {
+			$response = wp_remote_get( $style );
+			if ( ! is_wp_error( $response ) ) {
+				$styles[] = array(
+					'css'            => wp_remote_retrieve_body( $response ),
+					'__unstableType' => 'theme',
+				);
+			}
+		} else {
+			$file = get_theme_file_path( $style );
+			if ( is_file( $file ) ) {
+				$styles[] = array(
+					'css'            => file_get_contents( $file ),
+					'baseURL'        => get_theme_file_uri( $style ),
+					'__unstableType' => 'theme',
+				);
+			}
+		}
+	}
+}
```

Comment:
Is this chunk copied from `edit-form-blocks.php`? I wonder if it's maybe worth moving it into a new function in `wp-includes/block-editor.php` called e.g. `get_block_editor_styles` and calling that function here and in `edit-form-blocks.php`. That way if someone fixes a bug with this code in `edit-form-blocks.php` we'll get that fix here too.

Reply:
Yeah, that makes sense! I'll get onto it.

Reply:
Done!